src/pages/studies/StudyTableContainer.tsx

```diff
@@ -28,30 +27,24 @@ const mapState = (state: RootState): StudyTableContainerState => ({
 const connector = connect(mapState, mapDispatch);
 
 type PropsFromRedux = ConnectedProps<typeof connector>;
-export type PaginationType = {
-  pagination: TablePaginationConfig;
-};
 
-type Props = StudiesResults & PropsFromRedux & PaginationType;
+type Props = StudiesResults & PropsFromRedux & { total: number };
 
 const StudyTable = (props: Props) => {
-  const { pagination } = props;
-  const { current: currentPage, total: itemTotal = 0, pageSize: itemPerPage = 10 } = pagination;
-
   const tableData = generateTableData(props);
-  const pageRange = `${currentPage}-${itemTotal > itemPerPage ? itemPerPage : itemTotal}`;
+  const { total } = props;
 
   return (
     <div>
       <div className={styles.tableHeader}>
-        Showing <Text strong>{pageRange}</Text>
+        Showing <Text strong>{total}</Text>
```

Comment:
Should we just display : Showing {total} instead of Showing {total} out of {total} ?

Reply:
yes thats ok